luigi/tasks/genome_mapping/example_genome_locations.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+
+"""
+Copyright [2009-2018] EMBL-European Bioinformatics Institute
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+
+example_locations = {
```

Comment:
Do we have to worry about this across many genomes? I don't want to have to manage a file with hundreds of these (or have to fill it out hundreds of times).

Reply:
The `example_locations` list is needed to show a region with ncRNAs in the genome browser. This list is manually curated but maybe we can automate this (for example, query lucene to get any mapped URS for a taxid and retrieve its genome coordinates via RNAcentral API). Added for future discussion to https://github.com/RNAcentral/rnacentral-webcode/issues/145